cli/src/main/java/com/devonfw/tools/ide/tool/jasypt/Jasypt.java

```diff
@@ -103,7 +102,7 @@ private void runJasypt(String className) {
     }
 
     Java java = getCommandlet(Java.class);
-    java.runTool(null, arguments.toArray(new String[0]));
+    java.runTool(arguments.toArray(i -> new String[i]));
```

Comment:
It was not really necessary or required but I would consider this as improvement:
omitting optional parameters like `null` is a common best practice.
The `toArray` is often not understood in full detail:
When providing an array to the `toArray` method this method tries to fill in the data into that array.
However, when the size of the array does not fit, it will use reflection to determine the class of the array components and then again create a new array with the proper size. This is kind of waste since we create a new array that immediately is food for the garbage collector. Instead of `0` you could provide `arguments.size()`. However, in later Java versions this feature was added to provide a lambda function that gets the size and is then responsible to create the array with the proper given size. This is a generic pattern that can be used also with streams using filtering, mapping, etc. where you would not know the size.